src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -226,12 +227,40 @@ export default class OrganizationDiscover extends React.Component {
             />
           </Flex>
         </Header>
-        <Flex px={2}>
-          <Box w={[1 / 3, 1 / 3, 1 / 3, 1 / 4]}>
+        <DiscoverBody>
+          <Sidebar px={space(4)} w={[1 / 3, 1 / 3, 1 / 3, 1 / 4]} pb={space(3)}>
+            <SidebarHeader
+              mx={parseInt(space(4), 10) * -1}
```

Comment:
I think ideally we don't want to rely on js's type coercion here, instead I think having an extra parameter to `space()` to return an integer would be better.

Reply:
could this just be `space(4)`? what does the -1 pixel add?

Reply:
@lynnagara that's not adding -1px, it's multiplying the value of `space` by -1 (to get a -30px margin)

Reply:
Oh oops, read that wrong. In this case, I we shouldn't set 30px margin then unset it here with -30, just move the 30px margin to the nested element with the fields so it doesn't get applied to the sidebar header

Reply:
@lynnagara yeah that's fair. I actually suggested that @ayesha-omarali handle it this way, but that was before I realized we'd have to do the type coercion and all that.

@ayesha-omarali +1 on removing sidebar padding and sidebarHeader negative margin, and then adding margin or padding to Fieldset to compensate.

Reply:
Sounds good to me, thank you everyone updating now! 🙂